docs/builds.rst

```diff
@@ -186,6 +191,29 @@ key to the respective dictionaries of the build steps.
 Specifying the working directory for other steps works analogously to specifying the working directory for the
 configuration step (as seen above).
 
+Extra Paths
+-----------
+
+If you do not want to install your executable in ``@THIS_PREFIX_DIR@/bin`` you need to specify the
```

Comment:
Add some rationale why you would want/wouldn't want to do this. `<prefix>/bin` is standard for many UNIX packages, but not all, especially not all academic software that sometimes uses custom build systems.

Reply:
done